ast/ast.go

```diff
@@ -420,14 +420,26 @@ func (n *StringNode) GetValue() interface{} {
 	return n.Value
 }
 
-// String string value to text with quote if required
+// String string value to text with quote or literal header if required
 func (n *StringNode) String() string {
 	switch n.Token.Type {
 	case token.SingleQuoteType:
 		return fmt.Sprintf(`'%s'`, n.Value)
 	case token.DoubleQuoteType:
 		return fmt.Sprintf(`"%s"`, n.Value)
 	}
+
+	if strings.Contains(n.Value, "\n") {
```

Comment:
Looks good !!
But we may have to consider CRLF for windows .

Reply:
Yeah, you right.